nautobot/utilities/utils.py

```diff
@@ -503,5 +517,19 @@ def get_table_for_model(model):
     return get_related_class_for_model(model, module_name="tables", object_suffix="Table")
 
 
+def get_content_type_string(model):
+    """Return the the content type string given model.
+
+    Returns:
+        The name of the model in "{app_label}.{model_name}" format
+    """
+    if not inspect.isclass(model):
+        raise TypeError(f"model class {model} was passes as an instance!")
```

Comment:
```suggestion
    if not inspect.isclass(model):
        model = model.__class__
```

@glennmatthews @jathanism ?